packages/mux-video/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mux-elements/mux-video",
   "version": "0.3.0",
-  "type": "commonjs",
+  "type": "module",
```

Comment:
blocking: This is not accurate if our `main` is `dist/index.js`. We've been neglecting confirming different env usages, but we should probably have a followup effort there. Is there a reason you made this change in this PR?

Reply:
yes it was needed for web-test-runner to run mux-video's tests, otherwise it fails.

![SCR-20220314-e28](https://user-images.githubusercontent.com/360826/158201091-7aa081a2-98be-4036-b9bc-c926c46241cc.png)

would the esm bundle cause issues though? I think it's preferred actually

Reply:
it would potentially be a breaking change, since node/bundlers will start treating the IIFE code as ESM code.

On a related note, should `main` be pointing at the `iife` build or the `cjs` build?